apps/arm/ClawController.h

```diff
@@ -26,7 +26,10 @@ class ClawController : public ActuatorController {
 
     AnalogIn &r_forceSensor;
 
+<<<<<<< HEAD
```

Comment:
fix these merge conflict artifacts lol

Reply:
I rebased with alex's arm code at some point early on (I think), so thats why I have all this added. Not actually my code.

Reply:
These conflicts were probably introduced when you rebased, master would fail otherwise.